core/domain/topic_domain.py

```diff
@@ -965,6 +971,21 @@ def create_default_topic(
             constants.DEFAULT_LANGUAGE_CODE, 0,
             feconf.CURRENT_STORY_REFERENCE_SCHEMA_VERSION)
 
+    @classmethod
+    def _convert_subtopic_v2_dict_to_v3_dict(cls, subtopic_dict):
+        """Converts old Subtopic schema to the modern v3 schema. v3 schema
+        introduces the url_fragment field.
+
+        Args:
+            subtopic_dict: dict. A dict used to initialize a Subtopic domain
+                object.
+
+        Returns:
+            dict. The converted subtopic_dict.
+        """
+        subtopic_dict['url_fragment'] = None
```

Comment:
If you're doing a migration then maybe derive this from the subtopic name and truncate? Then we can modify it, and it never has to be None -- you can always make it required.